chapter_install/install.md

```diff
@@ -1,22 +1,74 @@
 # Installation
 :label:`chapter_installation`
 
-To get you up and running with hands-on experiences, we'll need you to set up with a Python environment, Jupyter's interactive notebooks, the relevant libraries, and the code needed to *run the book*.
+To get you up and running with hands-on experiences, we'll need you to set up with a Python environment, Jupyter's interactive notebooks, the relevant libraries, and the code needed to run the book.
 
-## Obtaining Source Codes
+## Installing Miniconda
 
-The source code package containing all notebooks is available at http://numpy.d2l.ai/d2l-en.zip. Please download it and extract it into a folder. For example, on Linux/macos, if you have both `wget` and `unzip` installed, you can do it through:
+To simplify the installation, we need to install [Miniconda](https://conda.io/en/latest/miniconda.html). The download link and file name are subject to changes, so please go the Miniconda website.
 
+```bash
+# For Mac users
+wget https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-x86_64.sh
+sudo sh Miniconda3-latest-MacOSX-x86_64.sh
+
+# For Linux users
+wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
+sudo sh Miniconda3-latest-Linux-x86_64.sh
```

Comment:
refer to 2.1.1.2
http://zh.d2l.ai.s3-website-us-west-2.amazonaws.com/chapter_prerequisite/install.html#Linux/macOS%E7%94%A8%E6%88%B7